### ISSUE TITLE
Can't create simio timer that has a size other than 3

### ISSUE THREAD

**Issue by tgtakaoka** [dlbeer/mspdebug#63]
Noticed that a size argument of `simio create time` seems ignored and always treated as 3.

```
$ mspdebug sim
...
(mspdebug) simio create timer TA7 7
Added new device "TA7" of type "timer".
(mspdebug) simio info TA7
Base address: 0x0160
IO address:   0x012e
IRQ0:         9
IRQ1:         8

TACTL:        0x0000
TAR:          0x0000
TAIV:         0x00

Channel  0, TACTL = 0x0000, TACCR = 0x0000
Channel  1, TACTL = 0x0000, TACCR = 0x0000
Channel  2, TACTL = 0x0000, TACCR = 0x0000
(mspdebug) 
```

**Comment by tgtakaoka** [comment]
This issue should be addressed by pull request #62.

**Comment by tgtakaoka** [comment]
Fixed.
